<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerDetailDialogue</class>
 <widget class="QDialog" name="CustomerDetailDialogue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Customer Details</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelCustomerName">
     <property name="text">
      <string>Customer Name:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditCustomerName"/>
   </item>
   <item>
    <widget class="QLabel" name="labelCustomerPhone">
     <property name="text">
      <string>Customer Phone:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditCustomerPhone"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
